### DISCUSSION TITLE
Support wechat pay and alipay

### DISCUSSION THREAD

**Discussion by fengyc** [lightsing/crates-mirror-ngx#26] [Ideas]
Hi, developers!

Thanks for the contribution.

I want to support the project and pay for a cup of coffee, but I don't have a patreon account. Could you add a wechat pay or alipay QRCode?

**Comment by lightsing** [comment]
Dear Feng,

Thanks for your support.

Afdian is supported, you can checkout at [https://afdian.net/@crates](https://afdian.net/@crates).

**Thank you!** 

Sincerely,

Lightsing and all of us at crates-io.cn
